src/wp-includes/default-filters.php

```diff
@@ -345,7 +345,7 @@
 add_action( 'wp_head', 'wp_generator' );
 add_action( 'wp_head', 'rel_canonical' );
 add_action( 'wp_head', 'wp_shortlink_wp_head', 10, 0 );
-add_action( 'wp_head', 'wp_custom_css_cb', 101 );
+add_action( 'wp_head', 'wp_custom_css_cb', -1 );
```

Comment:
This priority change will break some plugins, including Jetpack: https://wpdirectory.net/search/01H9VK1YVSPPD6Y9F3MZM4DEC6

Reply:
Feels like a blocker, is the recommend to revert?

Reply:
Possibly. Or a creative workaround might work: https://github.com/WordPress/wordpress-develop/pull/4824#discussion_r1322167495